Vungle/VungleRewardedVideoCustomEvent.m

```diff
@@ -105,15 +105,23 @@ - (void)vungleAdDidDisappear
     [self.delegate rewardedVideoDidDisappearForCustomEvent:self];
 }
 
-- (void)vungleAdWasTapped
+- (void)vungleAdTrackClick
 {
     MPLogAdEvent([MPLogEvent adTappedForAdapter:NSStringFromClass(self.class)], [self getPlacementID]);
     [self.delegate rewardedVideoDidReceiveTapEventForCustomEvent:self];
 }
 
-- (void)vungleAdShouldRewardUser
+- (void)vungleAdRewardUser
 {
-    [self.delegate rewardedVideoShouldRewardUserForCustomEvent:self reward:[[MPRewardedVideoReward alloc] initWithCurrencyAmount:@(kMPRewardedVideoRewardCurrencyAmountUnspecified)]];
+    // We should call `rewardedVideoShouldRewardUserForCustomEvent:reward:`
```

Comment:
Can you remove this comment since you are calling `rewardedVideoShouldRewardUserForCustomEvent:reward` in `rewardUser` method below?

Reply:
@ashinagawa Got it, thanks a lot. 🙂

Reply:
@ashinagawa Hi Aki, done. Thanks. 🙂